assets/html/scss/documenter/components/_clipboard.scss

```
@@ -0,0 +1,3 @@
+.copy-button {
+  float: right;
```

This should make sure that it's the same distance from top and right side of the box:

```suggestion
  float: right;
  top: -0.2rem;
```